internal/metrics/token_metrics.go

```diff
@@ -0,0 +1,69 @@
+// Copyright Envoy AI Gateway Authors.
+// SPDX-License-Identifier: Apache-2.0.
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package metrics
+
+import "time"
+
+// Status constants for request completion.
+const (
+	StatusSuccess = "success"
+	StatusError   = "error"
+)
```

Comment:
needs proper prefix then. like RequestStatusSuccess and RequestStatusError

Reply:
updated